### PR TITLE
Tests coverage and codeclimate config v2

### LINKED ISSUE #3: Add tests code coverage and code climate config v2

### PR DESCRIPTION
**Pull request by programmer4web**
fixes https://github.com/Neovici/cosmoz-theme-switcher/issues/3

### CODE REVIEW

**Comment by programmer4web** [comment]
@plequang This is ready for review.

File: analysis.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "schema_version": "1.0.0",
   "elements": [
     {
-      "description": "`<cosmoz-theme-switcher>` is a custom style theme manager\n\n### Usage\n\n    <cosmoz-theme-switcher></cosmoz-theme-switcher>",
+      "description": "`<cosmoz-theme-switcher>` is a custom style theme manager\n\n### Usage\n\n    <cosmoz-theme-switcher></cosmoz-theme-switcher>\n\neslint no-unused-vars: [\"warn\", { \"argsIgnorePattern\": \"event\" }]",
```
Comment (plequang) [inline analysis.json]: Hmm, where does this "eslint no-unused-vars" comes from at the end of the description ?

Will it be displayed somewhere ?
I've checked http://127.0.0.1:8081/components/cosmoz-theme-switcher/#/elements/cosmoz-theme-switcher and don't see it.

**Review by plequang — COMMENTED** [review]
Looks good overall but the description is strange in analysis.json

File: analysis.json (hunk 1 above)
Comment (programmer4web) [reply analysis.json]: two functions(themeError, themeLoaded) had unused param `event` I have removed that and generated a new analysis.json. added also arrow functions where needed and replaced var with const;

**Comment by programmer4web** [comment]
@plequang I have removed the unused parameters 'event'. This is ready for review.

**Review by plequang — APPROVED** [review]
LGTM

**Pull request merged**